pyproject.toml

```diff
@@ -34,7 +34,7 @@ pandas = [
     { version = ">=1.1,<1.6", python = ">=3.7,<3.11" },
     { version = ">=1.5,<1.6", python = ">=3.11" },
 ]
-pandera = ">=0.9.0,<1"
+pandera = ">=0.9.0,<0.16"
```

Comment:
Adding an upper bound is good for now, eventually we may want an explicit exclude on `0.16.0` such that we can pick up new versions without picking up this broken version. Looks like `pandera` is going to patch this one quickly